testing/README.md

```diff
@@ -1,4 +1,4 @@
-# perseus-testing
+# ../../testing
```

Comment:
Maybe just `# Testing`? 😄

Reply:
TBH, I think we can nuke this file since we're consolidating the various coverage reports into a single repo.